### ISSUE TITLE
One game works fine using Ruffle, another doesn't

### ISSUE THREAD

**Issue by dandareorg** [ruffle-rs/ruffle#4445]
Hello again, longtime no speak. I have only just got back into this after some major life changes this year.

So, my long-ago famous but now nearly dead site www.dan-dare.org and it's myriad of old Flash games:

This ancient game works fine with the very latest Ruffle update:
http://www.dan-dare.org/Dan%20Sonic/Dan%20Sonic%20Hi.htm

But sadly this slightly less ancient one doesn't:
http://www.dan-dare.org/FreeFun/Games/More/NewMarioFlash.htm

Given that my knowledge of web programming ends with basic HTML, I am unable to figure out the problem. Is there any chance someone there can work it out? It may be important to the overall Ruffle project. Then again, maybe not. I hope there is a solution though, as there are other great classic games in my site that probably won't work for the same reason as this one, whatever it is.

Thanks and regards,
Pete.

**Comment by flashliberty5467** [comment]
Thank your for testing ruffle on your website. @dandareorg 
The most helpful thing you can do is add games that work on your website the database provided by the Ruffle discord and any games that dont work file bug reports on the ruffle github and add the not working games to the database. 
These bug reports are important for improving ruffle

[https://docs.google.com/spreadsheets/d/1d4qn8OFLqLgD4R0Xt5SiS_P4ozlduHGbLE5RjlBOeRc/edit#gid=1190510451](url)

**Comment by Lord-McSweeney** [comment]
This seems to be working now

**Comment by dandareorg** [comment]
Yes, that is true. The most recent update of Ruffle that I added into my sites makes almost everythig work.

Very well done to you all!!

Yahoo Mail: Search, organise, conquer 
 
  On Thu, 11 Sept 2025 at 3:09, ***@***.***> wrote:   Lord-McSweeney left a comment (ruffle-rs/ruffle#4445)
This seems to be working now
